browser/resources/settings/brave_privacy_page/brave_personalization_options.ts

```diff
@@ -140,6 +158,35 @@ export class SettingsBravePersonalizationOptions extends SettingsBravePersonaliz
     this.browserProxy_.setStatsUsagePingEnabled(this.$.statsUsagePingEnabled.checked);
   }
 
+  onMetricsReportingChange_() {
+    const enabled = this.$.metricsReportingControl.checked;
+    this.browserProxy_.setMetricsReportingEnabled(enabled);
+  }
+
+  setMetricsReportingPref_(metricsReporting: MetricsReporting) {
```

Comment:
It looks like this was copied from the upstream code. It might be worth leaving comment about where it's from.

Reply:
Added